W3C/Android/Android.cs

```diff
@@ -44,8 +44,8 @@ static void Main(string[] args)
 
       // Initialize the remote Webdriver using BrowserStack remote URL
       // and desired capabilities defined above
-      AndroidDriver<AndroidElement> driver = new AndroidDriver<AndroidElement>(
-              new Uri("https://hub-cloud.browserstack.com/wd/hub"), capabilities);
+      AndroidDriver<AppiumWebElement> driver = new AndroidDriver<AppiumWebElement>(
```

Comment:
Don't change all of these. We want to have examples of other constructors as well

Reply:
done